packages/ra-ui-materialui/src/detail/EditView.tsx

```diff
@@ -60,8 +76,63 @@ export interface EditViewProps
     component?: ElementType;
     title?: string | ReactElement | false;
     sx?: SxProps;
+    unauthorizedView?: ReactElement;
+}
+
+const defaultActions = <EditActions />;
+
+function goBack() {
+    window.history.go(-1);
 }
 
+const DefaultUnauthorizedView = () => {
+    const translate = useTranslate();
+    return (
+        <Box
+            sx={theme => ({
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center',
+                justifyContent: 'center',
+                [theme.breakpoints.down('md')]: {
+                    padding: '1em',
+                },
+                fontFamily: 'Roboto, sans-serif',
+                opacity: 0.5,
+                '& h1': {
+                    display: 'flex',
+                    alignItems: 'center',
+                },
+            })}
+        >
+            <h1 role="alert">
+                <LockIcon
+                    sx={{
+                        width: '2em',
+                        height: '2em',
+                        marginRight: '0.5em',
+                    }}
+                />
+                {translate('ra-rbac.page.unauthorized')}
```

Comment:
wrong prefix: we're not in ra-rbac. Same below. 

Also, we must provide a default translation and populate the en and fr packages.